server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -198,10 +204,10 @@ public void canGetAllUsers() throws IOException {
   public void canGetUsersWithAge37() throws IOException {
     // Add a query param map to the context that maps "age" to "37".
     Map<String, List<String>> queryParams = new HashMap<>();
-    queryParams.put("age", Arrays.asList(new String[] {"37"}));
+    queryParams.put(UserController.AGE_KEY, Arrays.asList(new String[] {"37"}));
```

Comment:
This and many other places that follow were changing from magic numbers to specific values.

Reply:
We probably want to give names to the string `"37"`, or make that by calling `.toString()` on the value of `AGE_KEY`. Otherwise we've only kinda done half the job.